src/commands/help.js

```diff
@@ -8,45 +8,37 @@ module.exports = {
   execute(message, args) {
     const { commands } = message.client;
     //list of categories
```

Comment:
add a better description like
```suggestion
    // Here, we iterate through the list of the bots available commands. We first
    // get all the categories, and then...
```